r-pkg/R/elasticsearch_eda_funs.R

```diff
@@ -163,14 +168,30 @@ get_fields <- function(es_host
     httr::stop_for_status(result)
     resultContent <- httr::content(result)
```

Comment:
can you please explicitly state whether this is being parsed or you're just retrieving the text? i.e. make this `httr::content(result, as = 'parsed')`